luarules/gadgets/unit_stockpile_limit.lua

```diff
@@ -18,32 +18,27 @@ if gadgetHandler:IsSyncedCode() then
 	local CMD_INSERT = CMD.INSERT
 	local StockpileDesiredTarget = {}
 
-	local defaultStockpileLimit = 99
 	local unitStockpileLimit = {}
 
-	----------------------------------------------------------------------------
-	----------------------------------------------------------------------------
-
 	local GetUnitStockpile	= Spring.GetUnitStockpile
 	local GiveOrderToUnit	= Spring.GiveOrderToUnit
 
-	local canStockpile = {}
 	for udid, ud in pairs(UnitDefs) do
 		if ud.canStockpile then
-			canStockpile[udid] = true
-		end
-		if ud.weapons then
-			for i = 1, #ud.weapons do
-				local weaponDef = WeaponDefs[ud.weapons[i].weaponDef]
-				if weaponDef.stockpile and weaponDef.customParams and weaponDef.customParams.stockpilelimit then
-					unitStockpileLimit[udid] = tonumber(weaponDef.customParams.stockpilelimit)
+			unitStockpileLimit[udid] = 99	-- default stockpile limit
```

Comment:
`defaultStockpileLimit` should still be a variable, so there isn't a magic number